src/app/charts/charts-container.component.ts

```diff
@@ -17,7 +17,7 @@ import { ChartComponent } from './chart.component';
 })
 export class ChartsContainerComponent extends OnInit {
     private chartList: Array<String>;
-    private chartData: ChartData[];
+    private chartData: Array<ChartData>;
```

Comment:
Both are equally accepted. I changed this to see if it would fix linter complaints and it didn't. I just didn't both to change it back.